pkg/ebpf/common.go

```diff
@@ -106,6 +114,10 @@ func init() {
 	}
 }
 
+func isCentOS(platform string) bool {
```

Comment:
Is this the same logic that the main agent uses in their configuration setup? Would be good to see what they do for consistency

Reply:
I added something to read the `redhat-release` file as done here: https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/install_script.sh#L110

Reply:
Nice - just one last question. Does this file contain `centos`? (e.g. does this check need to be updated?)

Reply:
It does contain centos, example output: 
```
[vagrant@localhost ~]$ cat /etc/redhat-release
CentOS Linux release 7.6.1810 (Core)

```

Reply:
👍